presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -102,7 +102,20 @@ String
 
     Variable length character data with an optional maximum length.
 
-    Example type definitions: ``varchar``, ``varchar(20)``
+    Example type definitions: ``varchar``, ``varchar(20)``.
+
+    SQL supports simple and Unicode string literals:
+     - Literal string : ``'Hello winter !'``.
```

Comment:
```suggestion
     - Literal string : ``'Hello winter !'``
```
Either all, or none, of the items in a bulleted list need punctuation. If one requires punctuation then they all must. 

In this case, you end each item with inline code. Readers who copy your code examples may accidentally include the period at the end of the list item. Suggest removing the not wrong, but not needed punctuation to improve the reader's use of the doc. Remove the punctuation from every item in this list.